src/libs/ReportUtils.ts

```diff
@@ -5264,6 +5264,22 @@ function getChatByParticipantsAndPolicy(newParticipantList: number[], policyID:
     );
 }
 
+/**
+ * Attempts to find a report in onyx with the provided list of participants in given policy
```

Comment:
```suggestion
 * Attempts to find a report in onyx that is owned by ownerAccountID in a given policy
```

Not sure how to phrase this better, but the comment should be updated since we aren't passing "list of participants"